src/purse/collections.py

```diff
@@ -20,11 +20,12 @@
 import json
 from datetime import timedelta
 
-from aioredis import Redis
+from redis.asyncio import Redis
 from typing import Any, Dict, Type, Union, Tuple, Iterable, List
 from typing import TypeVar, Generic
 from collections.abc import Mapping, AsyncIterator
 from pydantic import BaseModel
+import msgspec
 
```

Comment:
So that's a faster alternative to Pydantic ?

Reply:
Then it needs to be a dependency

Reply:
That commit was not supposed to get into this MR, sorry. But yes, it is much faster and preserve some types such as byte in the conversion.

Reply:
great feature